### PR TITLE
Added guide to setup Firebase Crashlytics.

### PR DESCRIPTION
**Pull request by HRankit**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Fixed frontmatter on user's behalf.  Still needs to be reviewed.

File: guide/english/android-development/firebase/firebase-crashlytics/setup/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Setting it up with Android Studio
+
+Once you have added firebase to your project, you need to perform additional steps such as adding the relevant dependencies and so on. You can find out more information and sample code from the [documentation](https://firebase.google.com/docs/crashlytics/get-started#android).
+
+**Add Gradle Dependencies**
+
+In your `app-level build.gradle` file, add the following
+
+```java
```
Comment (thecodingaviator) [inline guide/english/android-development/firebase/firebase-crashlytics/setup/index.md]: Change spacing to 2-spaces per indent level

File: guide/english/android-development/firebase/firebase-crashlytics/setup/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+apply plugin:  'io.fabric'
+dependencies {
+    implementation 'com.google.firebase:firebase-core:16.0.4'  // Add dependency 
+    implementation 'com.crashlytics.sdk.android:crashlytics:2.9.5'**
+}
+```
+In your `project-level build.gradle` file, add the following
+
+```java
```
Comment (thecodingaviator) [inline guide/english/android-development/firebase/firebase-crashlytics/setup/index.md]: Here too

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@thecodingaviator This PR has gone stale.

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**